### ISSUE TITLE
Deprecated Features

### ISSUE THREAD

**Issue by bebaoboy** [maheshmnj/navbar_router#49]
I want to report some deprecated features on Flutter 3.22.0

`animated_navbar.dart`.

```
'onBackground' is deprecated and shouldn't be used. Use onSurface instead. This feature was deprecated after v3.18.0-0.1.pre.
```

```
'MaterialStateProperty' is deprecated and shouldn't be used. Use WidgetStateProperty instead. Moved to the Widgets layer to make code available outside of Material. This feature was deprecated after v3.19.0-0.3.pre.
```

**Solution**
Replace deprecated features with new ones.

**Comment by maheshmnj** [comment]
Fixed should be available in upcoming release

**Comment by maheshmnj** [comment]
Fixed and released in [v0.7.5](https://pub.dev/packages/navbar_router/changelog)
